src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -163,6 +163,7 @@ docker run -it --net=host --rm --gpus=all nvcr.io/nvidia/tritonserver:${RELEASE}
 
 ```bash
 genai-perf \
+  profile \
   -m gpt2 \
   --service-kind triton \
   --backend tensorrtllm \
```

Comment:
nit: I feel like placing subcommand on the same line as the genai-perf command makes it easier to notice what subcommand we are in (also helps to distinguish from other flags).
```suggestion
genai-perf profile \
  -m gpt2 \
  --service-kind triton \
  --backend tensorrtllm \
```

Reply:
Sure, done for all the docs. :)